python/ray/train/v2/_internal/execution/worker_group/worker.py

```diff
@@ -90,6 +84,17 @@ def _repr(self) -> str:
     def __repr__(self) -> str:
         return self._repr
 
+    def execute_async(self, fn: Callable[..., T], *fn_args, **fn_kwargs) -> ObjectRef:
```

Comment:
why we do this

Reply:
Oh I refactored this originally because I needed to call this logic without calling the `WorkerGroup.execute...` methods directly, but ended up discarding that change. This felt a bit cleaner to still have here for the existing calls, so I kept it.

Reply:
I see. I thought this was unused.